docs/sphinx/source/contributing/style_guide.rst

```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
 status box at the bottom of the pull request.
 
 
+.. _references:
+
+References
+----------
+All pertinent information within a docstring should include a proper reference.
+In pvlib-python, we are transitioning to a standardised referencing system. We
```

Comment:
```suggestion
In pvlib-python, we 
```

This page's scope is the reference style. The policy for what references are appropriate and where they are needed should be part of the Contributing guide, I think.

Reply:
Good point

Reply:
Still not worth mentioning the transitioning? I don't have a strong preference to include it, but I just thought it would explain why existing code that contributors see might not follow the recommended style. Still happy to delete that if you think so though 👍🏾

Reply:
Is it still not worth mentioning the transitioning bit?
I don't have a strong preference to include it, but I just thought it would explain why existing code that contributors see might not follow the recommended style. Still happy to delete that if you prefer though

Reply:
That's also a good point. "pvlib-python is transitioning to a standard reference format." Does this work?